gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -364,7 +364,9 @@ LANG.Styles = {
 		chat.AddText(warn_color, text)
 	end,
 
-	[MSG_CHAT_PLAIN] = chat.AddText
+	[MSG_CHAT_PLAIN] = chat.AddText,
+
```

Comment:
why an empty line? Previously, it made sense (because function declaration), but here it doesn't

Reply:
the different print types (`MSTACK` and `CHAT`) always had an empty line between them